{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -46,10 +46,8 @@
 # on_rtd is whether we are on readthedocs.org
 on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    import sphinx_rtd_theme
+if not on_rtd:  # only set the theme if we're building docs locally
     html_theme = 'sphinx_rtd_theme'
-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
@ionelmc Sounds good to me :+1: It's strange that the Sphinx seems to have done a 180° on inclusion of `sphinx_rtd_theme`…